### PR TITLE
Improve GLTF export and add node script (`npm run model`) to export specific models

### PR DESCRIPTION
**Pull request by Supalosa**
NOTE: I changed `loadFrame` to not flip the Z coordinate... it seems to improve the animation export but I have no idea if it broke anything else.

Gif recorded from a GLTF viewer (https://gltf-viewer.donmccurdy.com/) from a Jad model exported by this tool.

jad

### CODE REVIEW

File: src/cacheReader/loaders/ModelLoader.js [hunk 1]
```diff
@@ -356,7 +356,7 @@ export class ModelDefinition {
 	loadFrame(model, frame) {
 		let verticesX = [...model.vertexPositionsX];
 		let verticesY = [...model.vertexPositionsY];
-		let verticesZ = model.vertexPositionsZ.map(z => -z);
+		let verticesZ = [...model.vertexPositionsZ];
```
Comment (Supalosa) [inline src/cacheReader/loaders/ModelLoader.js:359]: @Dezinater  any comments on this? without it, the animations seem to flip the Z axis and render inside out.

Comment (Dezinater) [reply src/cacheReader/loaders/ModelLoader.js:359]: I think this does cause some issues with how non-animated models are exported. I forget exactly what this breaks but there would be 3 exports I'd test to make sure it's working correct, an animated model, a non animated model, and an Animaya animated model. If all of those export correctly without any special cases checking for their type then I think the code is correct

Comment (Supalosa) [reply src/cacheReader/loaders/ModelLoader.js:359]: 🤔  i might be able to flip the coordinate inside the gltf export code only then. Or at worst, make it flip when an argument is passed in.

To be fair I haven't even tried animaya or static models yet...

**Comment by Supalosa** [comment]
will resume discussion in the next PR

**Pull request closed without merging**